dashboard/src/components/SearchBox/SearchBox.js

```diff
@@ -45,6 +48,7 @@ const SearchInput = withStyles(theme => ({
 
 const styles = theme => ({
   root: {
+    overflow: "hidden",
     minWidth: 300,
     display: "flex",
     border: "1px solid",
```

Comment:
The clear button feels a bit uneven when compared to the filter icon on the left hand side of the container. Might be nice if the button was flush against the edge of the container.

...not a big deal though, not something we necessarily have to deal with right now.